packages/oauth/package.json

```diff
@@ -1,18 +1,17 @@
 {
   "name": "@cloudoperators/juno-oauth",
   "version": "1.2.7",
-  "description": "Authenticates user via OIDC",
   "author": "UI-Team",
-  "contributors": [
-    "Andreas Pfau"
-  ],
+  "description": "Authenticates user via OIDC",
+  "main": "build/oauth.es.js",
+  "module": "build/oauth.es.js",
```

Comment:
I think we need here to be and build/index.js

Reply:
same here

Reply:
Fixed